openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -787,8 +791,7 @@ def run(self, *, dry=False, verbose=True,
             softcore_alpha=alchem_settings.softcore_alpha,
             softcore_LJ_v2=alchem_settings.softcore_LJ_v2,
```

Comment:
This needs to be changed since we decided to create a new setting `softcore_LJ: Literal['gapsys', 'beutler']` (old setting: `softcore_LJ_vs: bool`. @IAlibay : Should we just change it here and keep the `softcore_LJ_v2` in the HTF, or should I also change it there?

Reply:
For now let's assume the signature of the HTF doesn't change.